.github/scripts/makeLambdaFunction.bash

```diff
@@ -0,0 +1,67 @@
+# !/bin/bash
+INITIALS_PROMPT="Please enter your first and last initial (or 'q' to quit)"
+NUM=2
+initials=$1
+
+while :
+  do
+    # get and validate user initials
+    printf "%s\n" "$INITIALS_PROMPT" 
+    while read initials && [ "$initials" != "q" ];
```

Comment:
The reason why the initials aren't showing up is because you have this while loop that will always run and overwrite the value of `$initials` retrieved on line 4.

If you look at the [script for the S3 bucket](https://github.com/North-Seattle-College/ad440-winter2022-tuesday-repo/blob/development/.github/scripts/makeReactBucket.bash) you'll see that this while loop is wrapped in an if statement that checks if `$initials` is null or not before running the loop.

You could either remove the while loop entirely or wrap it in an if statement to fix

Reply:
Removed from loop